### PR TITLE
Support for Rinkeby testnet

### LINKED ISSUE #181: Availability of the game on other testnets?
Currently Ropsten has a very low gas limit of `155040`, and yesterday there weren't any blocks for like an hour.

Additionally the community doesn't seem to be very active on it, and it's hard to find a faucet that works.

Is the game available on other testnets?

If not, IMO it would make sense to host another instance on Kovan or Rinkeby

### PR DESCRIPTION
**Pull request by hensha256**
Fixes #181

### CODE REVIEW

File: scripts/deploy_contracts.js [hunk 1]
```diff
@@ -71,9 +72,13 @@ async function deployContracts(deployData) {
...
     console.log(`Deploying Ethernaut.sol...`);
-    ethernaut = await Ethernaut.new(props)
-    console.log(colors.yellow(`  Ethernaut: ${ethernaut.address}`));
-    deployData.ethernaut = ethernaut.address;
+    try {
+      ethernaut = await Ethernaut.new(props)
+      console.log(colors.yellow(`  Ethernaut: ${ethernaut.address}`));
+      deployData.ethernaut = ethernaut.address;
+    } catch (err) {
+      console.log('Error deploying Ethernaut: ' + err)
```
Comment (frangio) [inline scripts/deploy_contracts.js:80]: Is it the intention to continue running or should the process be aborted if this fails?

**Review by frangio — APPROVED** [review]
Is receiving multiple mined logs for the same transaction expected? Is it a preexisting issue?



(It keeps going)

I was only able to test one level and I ran out of funds, but it looks like it's working well!

**Comment by frangio** [comment]
I'm thinking it would be nice to keep a ropsten version up just in case someone was mid-game and would like to continue on ropsten. We can publish it as `ropsten.ethernaut.openzeppelin.com`, and then include a link in the console message that says "you're on the wrong network!" if we detect ropsten. What do you think?

**Comment by hensha256** [comment]
Yeah the issue of it printing out the tx over and over is pre-existing in the solidity 0.5 version. I dont think it's an issue in the original version though so it shouldnt be too hard to pin down and stop in a future PR :)

File: scripts/deploy_contracts.js (hunk 1 above)
Comment (hensha256) [reply scripts/deploy_contracts.js:80]: I added this for debugging, I'll remove it

**Comment by hensha256** [comment]
Branch `ropsten-version` is ready to have ropsten support on a different subdomain

**Pull request merged**